File: lua/Ierusalimschy_3ed_book/ch03/ex05.lua
#!/usr/bin/lua52
-- Copyright (C) Khudyashev Ivan, 2018, bahek1983@gmail.com
-- Learning Lua Project
function isBooleanValue(val)
    return tostring(val or (val == false)) == "true"
end

print("value a = " .. tostring(a), isBooleanValue(a))
print("boolean value true", isBooleanValue(true))
print("boolean value false", isBooleanValue(false))

File: python3/ThePythonWorkshop/ch01/ex05.py
# Copyright (C) 2019-2021, Khudyashev Ivan, bahek1983@gmail.com

def doAction():
    x = 5
    print("now x = {}".format(x))
    y = 2
    print("now y = {}".format(y))
    print("x + x - y**2 = {}".format(x + x - y**2))
    x, y = 8, 5
    print("now (x, y) = ({}, {})".format(x, y))
    print("x // y = {}".format(x//y))

def main():
    doAction()

if __name__ == "__main__":
    main()
